lib/iproov_flutter.dart

```diff
@@ -36,8 +68,6 @@ class IProov {
       'token': token,
       'optionsJSON': json.encode(options)
     });
+    return completer.future;
```

Comment:
3. return `completer.future`. The net effect of this is that this method will return when the last subscription event has been received.

Makes the API more ergonomic because clients can call:

```dart
await iProov.launch((event) { ... });
// do something when complete
```